docs/source/docker_only.rst

```diff
@@ -0,0 +1,59 @@
+It is possible to develop with dallinger using only docker as a prerequisite.
+This guide goes through the necessary steps to achieve this.
+
+Set up services
+===============
+
+Start services needed by dallinger:
+
+.. code-block:: shell
+
```

Comment:
Also add(?)

```
docker network create -d bridge dallinger
```

Reply:
Might be worth mentioning the need to remove containers if they exist already:

```
docker rm dallinger_postgres
docker rm dallinger_redis
```

Reply:
Good points. I'll amend the instructions.